main.spec

```
@@ -0,0 +1,63 @@
+"""
+ * Copyright(c) 2021 to 2022 ZettaScale Technology and others
```

This can't be right!

I would have expected something like:
```suggestion
 * Copyright(c) 2024 Sven Trittler
```
I have agreed that what I do on Cyclone is copyright "ZettaScale Technology" (they pay for the vast majority of the time I spend on it, so it is totally fair). In your case, it could be that you also have some agreement like that with your employer.

The "and others" bit is technically not necessary when there is only one copyright holder. Any significant contribution by others should result in adding such an "other" to the header, if there's non-trivial but minor stuff, the "and others" is ok, given that all the details are available in the history. I tried to get them right, but it is such an easy thing to forget ...